manager/controllers/package_systems.go

```diff
@@ -65,7 +71,13 @@ func PackageSystemsListHandler(c *gin.Context) {
 		return
 	}
 
-	query := packageSystemsQuery(account, packageName)
+	var names []int
+	if err := packagesByNameQuery(packageName).Pluck("p.id", &names).Error; err != nil {
```

Comment:
To be correct, shouldn't we init transaction using db.Begin() and do both queries in it? It should ensure to have db in the same state for both queries. But for that case it shouln't be neccassarry.